kubernetes/services.html.markerb

```diff
@@ -69,6 +69,31 @@ spec:
       targetPort: 8080
 ```
 
+## Concurrency limits
+
+[Concurrency limits](https://fly.io/docs/reference/configuration/#services-concurrency) are used to limit the load on your application.
```

Comment:
In this case, with a cluster having one Machines, there's no load balancing and no auto stop or start, right?

Reply:
yep! but if your one instance hits its hard limit, we'll just fail requests to your app entirely